plugins/Files/js/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import createSagaMiddleware from 'redux-saga'
+import { createStore, applyMiddleware } from 'redux'
+import { Provider } from 'react-redux'
+import rootReducer from './reducers/index.js'
+import rootSaga from './sagas/index.js'
+import App from './containers/app.js'
+import { getWalletLockstate, getUploads, getFiles, setPath, getDownloads } from './actions/files.js'
+
+const sagaMiddleware = createSagaMiddleware()
+const store = createStore(
+	rootReducer,
+	applyMiddleware(sagaMiddleware)
+)
+sagaMiddleware.run(rootSaga)
+
+const rootElement = (
+	<Provider store={store}>
+		<App />
+	</Provider>
+)
+ReactDOM.render(rootElement, document.getElementById('react-root'))
+
+store.dispatch(getWalletLockstate())
+store.dispatch(getFiles())
+store.dispatch(setPath(''))
+store.dispatch(getUploads())
+
+const downloadStart = Date.now()
+
+setInterval(() => {
+	store.dispatch(getDownloads(downloadStart))
```

Comment:
The `downloadStart` construction is a little odd. As I understand, it's used to filter out downloads that were not initiated during the current session. Is that the behavior we want, though? In Firefox, I can always view all of the downloads I have ever made.